libraries/module-minifier/src/LocalMinifier.ts

```diff
@@ -44,8 +44,11 @@ export class LocalMinifier implements IModuleMinifier {
         : {}
     };
 
+    const { version: terserVersion } = require('terser/package.json');
+
     this._configHash = createHash('sha256')
       .update(LocalMinifier.name, 'utf8')
+      .update(`terser@${terserVersion}`)
       .update(serialize(terserOptions))
       .digest('base64');
```

Comment:
This is duplicated between the two. Put it in a utlity?

Reply:
Doing so doesn't really save much in terms of code footprint, and the two are slightly different. Might revisit later if there's a third one.